auditorium/views/decorators/with-authentication.js

```diff
@@ -8,9 +8,11 @@ function withAuthentication () {
       if (!state.authenticatedUser) {
         emit('offen:login', null)
         var authenticating = html`
-          <p class="loading">
-            ${__('Checking authentication...')}
-          </p>
+          <div class="row">
+            <div class="loading">
```

Comment:
Wrapping user facing text in non-semantic elements (i.e. `div`s) is bad for accessibility, so this will somehow need to stay a `p' or similar. Adding the `util` part of lit would add classes for controlling paddings and margins in such situations.

Reply:
ok had to remove `p` because of spacing issues - I'll undo that